tests/do_regtest.py

```diff
@@ -69,7 +70,7 @@ async def main() -> None:
     parser.add_argument("--skip_unittests", action="store_true")
     parser.add_argument("--skip_regtests", action="store_true")
     parser.add_argument("binary_dir", type=Path)
-    parser.add_argument("version")
+    parser.add_argument("version", nargs="?", default=None)
```

Comment:
Maybe change the default to an empty string as otherwise the `args.version.startswith("p")` below could fail.